packages/cli/src/transforms/toktx.ts

```diff
@@ -164,8 +165,13 @@ export const toktx = function (options: ETC1SOptions | UASTCOptions): Transform
 				const extension = texture.getURI()
 					? FileUtils.extension(texture.getURI())
 					: ImageUtils.mimeTypeToExtension(texture.getMimeType());
-				const inPath = tmp.tmpNameSync({ postfix: '.' + extension });
-				const outPath = tmp.tmpNameSync({ postfix: '.ktx2' });
+
+				const dir = `${tmp.tmpdir}/glTF-transform`;
+				if(!existsSync(dir)) mkdirSync(dir);
+				const tmpPrefix = `${textureIndex}`;
+				const tmpName = tmp.tmpNameSync({ dir: dir, prefix: tmpPrefix });
```

Comment:
Yes looks good! 

Do you think it would be doable to also include the original texture name here again - I briefly tried that but at the point is not (anymore?) available from ``texture.getName()`` and I wasnt sure if it would be worth the hazzle to try to pass it through to toktx (didn't want to introduce many more changes)

Reply:
Also would you prefer if the texture index was at the end of the name? 
I thought it makes it easier to search when dealing with a ton of textures and I know I want to check the texture at index 5

Reply:
The texture may not have `.getName()` or `.getURI()` defined in the file, this isn't too uncommon. If you're exporting to `.gltf`, glTF Transform will generate a reasonable file name (https://github.com/donmccurdy/glTF-Transform/pull/676/files) itself. But I think it's probably too much dependency to try to generate that same name here. I'd probably skip for this PR, but appending `FileUtils.basename(.getURI()) || .getName()` could be added later.